pkg/hubble/parser/seven/tracing.go

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Hubble
+
+package seven
+
+import (
+	"context"
+	"net/http"
+
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const traceparentHeader = "traceparent"
```

Comment:
I think it should be documented somewhere that trace ID will be extracted only if it's propagated following the [W3C trace context spec](https://www.w3.org/TR/trace-context/). This header in particular looks like a magic value, a link to the standard could be helpful here.